js/controllers/authors.js

```diff
@@ -0,0 +1,69 @@
+/* global app */
+/* global angular */
+
+app.controller('authorsCtrl', ['$scope', 'authorService', 'enableEditService', 'bookService', function($scope, authorService, enableEditService, bookService, $index) {
```

Comment:
are these a list of all the dependencies for this module? If so, is bookService a dependency? If not, can it be removed to clean up the code?

Reply:
yes, i thought it may have been needed at one point in the past but it has now been removed